slowmovie.py

```diff
@@ -105,15 +105,44 @@ def video_info(file):
         frameTime = 1000 / fps
 
         info = {
-            'frame_count': frameCount,
-            'fps': fps,
-            'duration': duration,
-            'frame_time': frameTime}
+            "frame_count": frameCount,
+            "fps": fps,
+            "duration": duration,
+            "frame_time": frameTime}
 
         videoInfos[file] = info
     return info
 
 
+# Returns the next video in the videos directory, or the first one if there's no current video
+def get_next_video(viddir, currentVideo=None):
+    # Only consider videos in the directory
+    videos = list(filter(supported_filetype, os.listdir(viddir)))
```

Comment:
```suggestion
    videos = sorted(list(filter(supported_filetype, os.listdir(viddir))))
```
Should put this back

Reply:
Thanks ✅